src/runtime/virtcontainers/sandbox.go

```diff
@@ -2413,17 +2413,17 @@ func (s *Sandbox) prepareEphemeralMounts(memoryMB uint32) ([]*grpc.Storage, erro
 			// if a tmpfs ephemeral mount is present
 			// update its size to occupy the entire sandbox's memory
```

Comment:
You probably need to update this comment, since the behaviour you implemented is slightly more subtle.